packages/babel-plugin-minify-constant-folding/__tests__/fixtures/member-access/expected.js

```diff
@@ -0,0 +1,7 @@
+'oof';
+void 0;
+({}).toString;
```

Comment:
This is unsafe — what would happen if `toString` was overridden to call `JSON.stringify`?

Reply:
https://github.com/babel/minify/pull/814#discussion_r178538385.

I really think we should clear https://github.com/babel/minify/issues/815 first.